src/crucible/aws/ecs.clj

```diff
@@ -0,0 +1,19 @@
+(ns crucible.aws.ecs
+  "Resources in AWS::ECS::*"
+  (:require [clojure.spec.alpha :as s]
+            [crucible.resources :refer [spec-or-ref defresource]]
+            [crucible.aws.ecs.task-definition :as td]
+            [crucible.aws.ecs.service :as svc]))
+
+(defn ecs [suffix] (str "AWS::ECS::" suffix))
+
+(s/def ::cluster-name (spec-or-ref string?))
```

Comment:
Although it is only one, I would move this into it's own namespace as well for consistency.